configs/lanl/config.MatchBoth_orig

```diff
@@ -49,7 +49,7 @@ baseDirectory = /dir/to/analysis/output
 # option:
 #    ./run_analysis.py config.analysis --generate \
 #         all,no_ocean,all_timeSeries
-generate = ['all']
+generate = ['all_climatology']
```

Comment:
@milenaveneziani, was this just for your testing or do the other analysis tasks not work for some reason?  If the former, let's leave this general.

Reply:
ah, sorry. forgot to change this!

Reply:
Oh, right.  I can do that.